src/Maths/math.vector.ts

```diff
@@ -3552,6 +3552,64 @@ export class Quaternion {
  * Class used to store matrix data (4x4)
  */
 export class Matrix {
+    private static _Use64Bits = true;
+
+    private static _TrackPrecisionChange = true;
+    private static _TrackedMatrices: Array<Matrix> = [];

```

Comment:
I honestly don't think we need this. The documentation should state that you have to pick your side at the beginning

Reply:
The problem are the static matrices: they are created right at start, before you had a chance to change the `Use64Bits` flag.

See my explanations above and in my comment below.